modules/microshift-config-yaml.adoc

```diff
@@ -4,81 +4,43 @@
 
 :_content-type: CONCEPT
 [id="microshift-config-yaml_{context}"]
-= Using a YAML configuration file
+= Configuring {product-title}
 
-Configure your {product-title} using a YAML file. The values specified in your configuration file are used when you execute a command.
-// Q: can these values be overriden with CLI flags?
```

Comment:
Technically, yes, but we do not want to support that long-term and will likely remove it soon.

Reply:
"Using a YAML configuration file" is Docs-style correct. "Configuring MicroShift" is vague and hard to find, especially if other methods are added later.

Reply:
Restored